File: Data Structures & Algorithms/counting-bits/submission-2.java
class Solution {
    public int[] countBits(int n) {
        int[] ans = new int[n + 1];
        for(int i=0;i<=n;i++){
            int k = i;
            int res = 0;
            while(k != 0){
                res += k % 2;
                k = k >> 1;
            }
            ans[i] = res;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/anagram-groups/submission-2.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        mp = {}
        ans = []
        for i in strs:
            k = [0] * 26
            for j in i:
                idx = ord(j) - ord('a')
                k[idx] += 1
            if tuple(k) in mp:
                mp[tuple(k)].append(i)
            else:
                mp[tuple(k)] = [i]
        for k,v in mp.items():
            ans.append(v)
        return ans